fem/qspace.hpp

```diff
@@ -30,13 +30,16 @@ class QuadratureSpaceBase
    Mesh &mesh; ///< The underlying mesh.
    int order; ///< The order of integration rule.
    int size; ///< Total number of quadrature points.
+   int ne; ///< Actual number of entities
    mutable Vector weights; ///< Integration weights.
    mutable long nodes_sequence = 0; ///< Nodes counter for cache invalidation.
 
-   /// @brief Entity quadrature point offset array, of size num_entities + 1.
+   /// @brief Entity quadrature point offset array. Supports a constant
```

Comment:
Sorry, another Doxygen nitpick. Can you move everything from the second sentence on ("Supports a constant...") to a new paragraph, separated by a blank line?